app/models/user.rb

```diff
@@ -39,9 +39,9 @@ def self.search(term)
 
   # avatar returns the user's avatar, or a default avatar if the user has not set one. It cannot return nil.
   def avatar
-    return "https://splits.io/logo.svg" if read_attribute(:avatar).nil?
+    return 'https://splits.io/logo.svg' if self[:avatar].nil?
```

Comment:
Looks like `self[:avatar]` will error if nil, see: https://github.com/rubocop-hq/rails-style-guide/issues/155

Reply:
Rubocop made it sound identical and I was trying to squash a warning, so I'll just revert this.